### DISCUSSION TITLE
How is everyone using container tabs and profiles?

### DISCUSSION THREAD

**Discussion by dParikesit** [zen-browser/desktop#1984] [Q&A]
IIRC, profiles are separate browser instances with different cache, sessions, etc (just like in ARC browser), so profiles and containers are somewhat similar. So yeah I'm wondering how is everyone using container tabs and profiles? Do you have a clear separation in your workflow on when to use containers and when to use profiles? Do you also have a specific case on when to use workspaces?

**Comment by RealDishwash** [comment]
> IIRC, profiles are separate browser instances with different cache, sessions, etc (just like in ARC browser), so profiles and containers are somewhat similar. So yeah I'm wondering how is everyone using container tabs and profiles? Do you have a clear separation in your workflow on when to use containers and when to use profiles? Do you also have a specific case on when to use workspaces?

I use Workspaces to seperate between my personal/university stuff and then i have container tabs for those as well so i don't really use profiles as much as i used to on chromium browsers. 

Containers let you keep everything in the same window and with the same extensions while profiles are a whole new instance of the browser with a different set of extensions/themes you can choose.

**Comment by Mircobrb** [comment]
Same as @RealVishy I mostly use workspaces, no great use for profiles as of now.

I have set up 3 different workspaces: Personal, Gaming, Work.

Personal and Gaming use no container, while Work has a default container for all tabs, so I can have access on all tools like Gmail, Figma etc in each specific workspace with a specific email and user.

Could be cool to integrate the profiles in the workspaces just like arc, but its probably overkill for most use cases

**Comment by Keltirion** [comment]
I have 4 workspaces, 3 for work depends on client and one for personal stuff. Each workspace has its own container. Due to lack of bookmarks per workspace my personal bookmarks do not have a place in there. I'm thinking of moving my personal workspace to different profile just for that reason. As work bookmarks i can share across three workspaces.

**Comment by mikedlr** [comment]
After experiments and transferring quite a bit from my Firefox setup: 
* most workspaces match with one (default) container but sometimes use no container
* some workspaces have multiple different containers (e.g. so that multiple mail accounts can be visible together.
* Firefox multi account containers manages the different containers
* I do not use the dedicated site feature in multi-account-containers.
